config/locales/pt-BR.yml

```diff
@@ -518,6 +518,7 @@ pt-BR:
       accounts: Contas
       balance: 'Balanço:'
       categories: Serviços Oferecidos
+      change_your_image:
```

Comment:
is this a problem @sseerrggii ? I guess this will be displayed as an empty string which might troublesome.

Reply:
I don't think so @sauloperez, we are using `i18n.fallbacks = true` so it should render the default_locale.